diff --git a/Makefile b/Makefile
index 9a94322..7f3ebb6 100644
--- a/Makefile
+++ b/Makefile
@@ -1,11 +1,13 @@
-DOCKER_USER:=pierres
-DOCKER_ORGANIZATION=archlinux
-DOCKER_IMAGE:=base
+DOCKER_USER:=attilabogar
+DOCKER_ORGANIZATION=attilabogar
+DOCKER_IMAGE:=archbase-x86_64
 
 rootfs:
 	$(eval TMPDIR := $(shell mktemp -d))
 	env -i pacstrap -C /usr/share/devtools/pacman-extra.conf -c -d -G -M $(TMPDIR) $(shell cat packages)
 	cp --recursive --preserve=timestamps --backup --suffix=.pacnew rootfs/* $(TMPDIR)/
+	echo -e '\n[multilib]\nInclude = /etc/pacman.d/mirrorlist' >> $(TMPDIR)/etc/pacman.conf
+	cp -p /etc/pacman.d/mirrorlist $(TMPDIR)/etc/pacman.d/
 	arch-chroot $(TMPDIR) locale-gen
 	arch-chroot $(TMPDIR) pacman-key --init
 	arch-chroot $(TMPDIR) pacman-key --populate archlinux
